integration/spark2/src/main/scala/org/apache/spark/sql/hive/CarbonRelation.scala

```diff
@@ -191,6 +191,14 @@ case class CarbonRelation(
         }
       }
     }
+    else if (carbonTable.isExternalTable) {
```

Comment:
add check in above code for normal table, no need to check for tablestatus file as extrenal table tablestatus will not be present